test/conformance/broker_tracing_test.go

```diff
@@ -0,0 +1,26 @@
+// +build e2e
+
+/*
+Copyright 2019 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```

Comment:
Also linefeed here, and in the header of other new files.

Reply:
Done.